configuration/openmrs/i18n/appointments/locale_fr.json

```diff
@@ -0,0 +1,149 @@
+  "MESSAGE_DIALOG_OPTION_COPY": "Copiez l'erreur",
+  "MESSAGE_DIALOG_OPTION_OKAY": "Ok",
+  "ADD_SERVICE_APPOINTMENT_TYPE": "Ajouter un type de rendez-vous",
+  "ADD_NEW_TYPE": "Ajouter un nouveau type",
+  "PLACEHOLDER_SERVICE_TYPE_NAME": "Nom du type de service",
+  "ADD": "Ajouter",
+  "DUPLICATE_SERVICE_APPOINTMENT_TYPE_ERROR_MESSAGE": "Ce type de rendez-vous de service existe déjà.",
+  "APPOINTMENT_SERVICE_DELETE_SUCCESS_MESSAGE_KEY": "Service supprimé.",
+  "APPOINTMENT_SERVICE_TYPE_DELETE_CONFIRMATION_DIALOG_MESSAGE_KEY": "Des rendez-vous attachés à ce type de service sont déjà sauvés pour une date future, veuillez les annuler avant de supprimer ce type de service. Une fois le type de service supprimé, vous ne pourrez plus voir les rendez-vous qui y sont attachés.",
+  "ADD_NEW_APPOINTMENT": "Ajouter un nouveau rendez-vous",
+  "APPOINTMENT_CREATE_SEARCH_PATIENT": "Recherche de patients",
+  "PLACEHOLDER_APPOINTMENT_CREATE_SEARCH_PATIENT": "Nom ou ID du patient",
+  "APPOINTMENT_CREATE_PATIENT_NAME": "Patient",
+  "APPOINTMENT_SERVICE": "Service",
+  "PLACEHOLDER_SERVICE": "Sélectionnez un service",
+  "PLACEHOLDER_SERVICE_TYPE": "Sélectionnez un type de service",
+  "PLACEHOLDER_CREATE_PROVIDER": "Sélectionnez un soignant",
+  "APPOINTMENT_DATE": "Date",
+  "APPOINTMENT_START_TIME_KEY": "Heure de début",
+  "APPOINTMENT_END_TIME_KEY": "Heure de fin",
+  "PLACEHOLDER_APPOINTMENT_TIME_FORMAT": "Entrez l'heure en hh: mm am / pm",
+  "APPOINTMENT_CREATION_DATE_WARNING": "La date choisie est en dehors des dates de disponibilité du service",
+  "APPOINTMENT_CREATION_START_TIME_WARNING": "L'heure de début est en dehors des heures de disponibilité du service",
+  "APPOINTMENT_CREATION_END_TIME_WARNING": "L'heure de fin est en dehors des heures de disponibilité du service",
+  "APPOINTMENT_CREATION_OUT_OF_RANGE_WARNING": "Les horaires sélectionnés sont en dehors des horaires disponibles pour ce service",
+  "APPOINTMENT_CREATE_WALK_IN": "Sans rendez-vous",
+  "APPOINTMENT_CREATE_NOTES": "Notes",
+  "PLACEHOLDER_CREATE_NOTES": "Entrer des notes de rendez-vous",
+  "APPOINTMENT_CREATE_SAVE": "Valider",
+  "APPOINTMENT_CREATE_CANCEL": "Annuler",
+  "APPOINTMENT_CREATE_SLOT": "Slots",
+  "APPOINTMENT_CREATE_BOOKED": "Booked",
+  "APPOINTMENT_CREATE_AVAL": "Available",
+  "APPOINTMENT_CREATE_OUTOF": "out of",
+  "APPOINTMENT_PATIENT_ID": "Patient ID",
+  "APPOINTMENT_PATIENT_NAME": "Patient Name",
+  "APPOINTMENT_CHECKIN_TIME_KEY": "Check in time",
+  "APPOINTMENT_SERVICE_TYPE_FULL": "Service Appointment Type",
+  "APPOINTMENT_WALK_IN": "Walk In",
+  "APPOINTMENT_STATUS": "Status",
+  "APPOINTMENT_COMMENTS": "Comments",
+  "CHECK_IN_APPOINTMENT": "Check-in",
+  "COMPLETE_APPOINTMENT": "Complete",
+  "MISSED_APPOINTMENT": "Missed",
+  "EDIT_APPOINTMENT": "Edit",
+  "CANCEL_APPOINTMENT": "Cancel",
+  "CURRENT_DATE": "Aujourd'hui",
+  "DAY": "Jour",
+  "WEEK": "Semaine",
+  "TOTAL_KEY": "Total",
+  "MISSED_KEY": "missed",
```

Comment:
Could you replace by "Manqué" or "manqué" .

Reply:
[Done](https://github.com/mekomsolutions/bahmni-config-easydoc/pull/6/commits/427449b1d84cce0b217ef35f341a63c96db84f90)